docs/reference/asyncapi-rules.md

```diff
@@ -288,6 +330,12 @@ Server URL should not point at example.com.
 
 **Recommended:** No
 
+### asyncapi-server-security
+
+Server `security` values must match a scheme defined in the `components.securitySchemes` object. It also checks if there are `oauth2` scopes that have been defined for the given security.
+
```

Comment:
Do you not want examples here?

Reply:
I didn't see that comment 😆 Added. Also added for `asyncapi-operation-security`.